source/efficient_mode/EfficientSymbiont.h

```diff
@@ -91,7 +91,7 @@ class EfficientSymbiont: public Symbiont {
    * Purpose: Mutating the efficiency of an efficient symbiont based upon the config
    * setting for mutation size.
    */
-  void mutate(){
+  void mutate(std::string mode = "vertical"){
     Symbiont::mutate();
```

Comment:
We need to set up the mutate function to work for horizontal transmission mutation as well in these subclasses